service/history/execution/mutable_state_util_test.go

```diff
@@ -24,13 +24,100 @@ import (
 	"testing"
 	"time"
 
+	fuzz "github.com/google/gofuzz"
 	"github.com/stretchr/testify/assert"
 
 	"github.com/uber/cadence/common"
 	"github.com/uber/cadence/common/clock"
+	"github.com/uber/cadence/common/persistence"
+	"github.com/uber/cadence/common/testing/testdatagen"
 	"github.com/uber/cadence/common/types"
 )
 
+func TestCopyActivityInfo(t *testing.T) {
+	for i := 0; i < 100; i++ {
+		t.Run("test CopyActivityInfo Mapping", func(t *testing.T) {
+			f := fuzz.New().Funcs(testdatagen.GenHistoryEvent).NilChance(0.2)
```

Comment:
to preserve repeatability of tests let's use the fuzzer with fixed seed. `NewWithSeed`

Reply:
👍  will do

Reply:
I may vote for:
- use `time.Now()` as a seed
- `t.Log` that seed so it can be recreated

fuzz tests kinda _should_ keep changing, to hit new edge cases you didn't expect.  otherwise you're just invisibly hardcoding an unknown set of assumptions, hidden behind `rand()`.

Reply:
I like that idea of changing seed but at least have it in the logs so we can reproduce